### PR TITLE
Update README.md to use dep ensure rather than go get

### LINKED ISSUE #111: update README.md - dont need go get -t for local binary build

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes #111

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ $ docker run -p 8080:8080 taco
 ### Build for the local OS
 ```shell
 % cd cmd/tacod
-% go get -t
+% dep ensure -vendor-only
```
Comment (aaron-collier) [inline README.md]: Is the `-vendor-only` flag necessary?

Comment (jcoyne) [reply README.md]: Yes, otherwise it’ll recalculate the dependencies rather than use what we have tested locally, which is specified by the lock file.

Comment (cmh2166) [reply README.md]: No, this is exactly the opposite of what we've been working through.

Comment (cmh2166) [reply README.md]: Justin, have you done #94 yet? I think the rest of the group has - and I don't know if what issues you are having due to your setup are occurring for others. I'd say close this PR until you've gone through #94 and we have some more concrete, shared knowledge on usage of dep.

Comment (jcoyne) [reply README.md]: Here is some evidence of why `dep ensure` gives a user a version that is not the same as what we've developed on: https://github.com/sul-dlss-labs/taco/pull/151

**Pull request closed without merging**